### PR TITLE
Gets the count of unread messages from MessageBus

### PR DESCRIPTION
**Pull request by selvakumaar5496**
Signed-off-by: Selvakumar <selva.k.nambi@seagate.com>

### CODE REVIEW

File: py-utils/src/utils/message_bus/message_bus_client.py [hunk 1]
```diff
@@ -90,6 +91,16 @@ def delete(self):
         client_id = self._get_conf('client_id')
         self._message_bus.delete(client_id, message_type)
 
+    def get_unread_count(self, consumer_group: str = None):
```
Comment (ujjwalpl) [inline py-utils/src/utils/message_bus/message_bus_client.py:94]: This interface should be wrapped by consumer class with consumer_group predefined.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/utils/message_bus/message_bus_client.py (hunk 1 above)
Comment (sachinpunadikar) [reply py-utils/src/utils/message_bus/message_bus_client.py:94]: @ujjwalpl  if we make that change, then we are always going to get the unread count for the default consumer_group. Getting the unread count for another consumer_group from consumer side will not be possible. I hope this is as expected.

Comment (ujjwalpl) [reply py-utils/src/utils/message_bus/message_bus_client.py:94]: Yes! This is what we need. We need for the current consumer group only.

Comment (selvakumaar5496) [reply py-utils/src/utils/message_bus/message_bus_client.py]: done, as suggested.

**Review by ujjwalpl — APPROVED** [review]

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**